metis/bin/metis_client

```diff
@@ -1125,10 +1137,19 @@ EOT
       end
     end
 
+    NON_FILE_MATCH=/[\]\[<>:;,?"*\|\/\x00-\x1f]+/
```

Comment:
This also includes `[]` although Metis allows those, because they seemed to give me trouble.